cpp/src/slice2js/Gen.cpp

```diff
@@ -673,11 +673,13 @@ Slice::Gen::~Gen()
     if (_jsout.isOpen() || _useStdout)
     {
         _jsout << '\n';
+        _jsout.close();
```

Comment:
do we need to call `close` when `_useStdout` is true? previously we didn't.

Reply:
We don't need to, but we don't really need to for the files either. All the handles get closed when the process exits. It's not an error to do it, so I think it's fine here.